core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockletDataRefNodeWrapper.java

```diff
@@ -132,13 +132,48 @@ public DimensionRawColumnChunk getDimensionChunk(FileHolder fileReader, int bloc
   public MeasureRawColumnChunk[] getMeasureChunks(FileHolder fileReader, int[][] blockIndexes)
       throws IOException {
     MeasureColumnChunkReader measureColumnChunkReader = getMeasureColumnChunkReader();
-    return measureColumnChunkReader.readRawMeasureChunks(fileReader, blockIndexes);
+    MeasureRawColumnChunk[] measureRawColumnChunks =
+        measureColumnChunkReader.readRawMeasureChunks(fileReader, blockIndexes);
+    updateMeasureRawColumnChunkMinMaxValues(measureRawColumnChunks);
+    return measureRawColumnChunks;
   }
 
   @Override public MeasureRawColumnChunk getMeasureChunk(FileHolder fileReader, int blockIndex)
       throws IOException {
     MeasureColumnChunkReader measureColumnChunkReader = getMeasureColumnChunkReader();
-    return measureColumnChunkReader.readRawMeasureChunk(fileReader, blockIndex);
+    MeasureRawColumnChunk measureRawColumnChunk =
```

Comment:
Does it require to change at runtime? is it possible to update those while calling `CarbonUtil.updateMinMaxValues`

Reply:
here I am not calling CarbonUtil.updateMinMaxValues. I am just reversing the min and max values of measures based on oldStore flag. Thsi can be done only during runtime because this min max is at the page level which is read only at the the page level pruning